src/lib/components/profile/chip/Chip.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import cancelIcon from '$lib/icons/cancel.svg'
+  import { expand, shrink } from './expand'
+
+  export let text: string
+  export let key: string
+  export let animation: boolean = true
```

Comment:
```suggestion
  export let animation = true
```
Creo que no hace falta especificar el tipo en este caso